src/Microsoft.VisualBasic/src/Microsoft/VisualBasic/ControlChars.vb

```diff
@@ -0,0 +1,22 @@
+' Copyright (c) Microsoft. All rights reserved.
+' Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+Namespace Microsoft.VisualBasic
+    ' Contants for the Control Characters
+    Public NotInheritable Class ControlChars
+
+        Public Const CrLf As String = ChrW(13) & ChrW(10)
+        Public Const NewLine As String = ChrW(13) & ChrW(10)
```

Comment:
@stephentoub: NewLine is a constant, and is embedded inside the assembly by the VB compiler. This means that if the value is platform-dependent ("\r\n" on Windows, but "\n" on Unix) you'll get different behavior depending on where you compiled your app, not depending on where you run it, which is not what most people (everybody?) expect.